workers/unity/Packages/io.improbable.gdk.core/Worker/NewWorld/Bootstrap.cs

```diff
@@ -0,0 +1,284 @@
+﻿using System;
```

Comment:
Ignore this whole file, nothing much changed, just adapt worker and world constructors